tests/desktop/consumer_pages/test_details_page.py

```diff
@@ -14,6 +14,9 @@ class TestDetailsPage(BaseTest):
     @pytest.mark.sanity
     @pytest.mark.nondestructive
     def test_that_application_page_contains_proper_objects(self, mozwebqa):
+        if '-dev' in mozwebqa.base_url:
+            pytest.xfail("Bug 1242648 - Support info doesn't show up on an empty profile")
```

Comment:
It should be essentially the same, in fact with pytest-mozwebqa there's only one source of base URL so it's even more reliable. Something like the following should do it:

```python
@pytest.mark.xfail('-dev' in pytest.config.getvalue('baseurl'),
                   reason="Bug 1242648 - Support info doesn't show up on an empty profile")
def test_xfail_dev(self, mozwebqa):
    ...
```